Idle_Game/app/src/main/java/com/example/idle_game/ui/theme/Color.kt

```diff
@@ -38,6 +38,10 @@ val surfaceContainerLight = Color(0xFFF0EEE6)
 val surfaceContainerHighLight = Color(0xFFEAE8E1)
 val surfaceContainerHighestLight = Color(0xFFE4E2DB)
 
+val Purple40 = Color(0xFF6650a4)
+val PurpleGrey40 = Color(0xFF625b71)
+val Pink40 = Color(0xFF7D5260)
+val bitcoinBackground = Color(0xFFF7931A)
```

Comment:
val Purple40 = Color(0xFF6650a4)
val PurpleGrey40 = Color(0xFF625b71)
val Pink40 = Color(0xFF7D5260)
val bitcoinBackground = Color(0xFFF7931A)
Werden alle in keinem Theme verwendet, müssen also wieder entfernt werden.
Lege die Farben für Elemente nicht statisch fest. Sie sollen bei runtime von den Nutzereinstellungen beinflusst werden

Reply:
Ist gefixt